app/views/capture/index2.html.erb

```diff
@@ -188,6 +240,15 @@
                   <a rel="tooltip" title="Scale video" class="btn btn-default" onClick="$W.scale_h = parseFloat(prompt('Enter a horizontal scaling factor (default is 1):'))"><i class="fa fa-expand"></i></a>
                 </div>
                 <p style="padding: 1rem">Note: Use the tools above to make adjustment of the capturing. To adjust the sampling yellow bar location, go back to the previous step.</p>
+
+                <div>
+                <p><small>Using calibration:</small></p>
+                <select name="spectrum[calibration_id]" class="select-calibration select-calibration-capture span2">
+                  <%= render partial: 'capture/calibrations', locals: { calibrations: @calibrations, calibration: @calibration } %>
```

Comment:
Wow very cool! Can you screenshot this as well, for documentation?

Reply:
It's still not functioning as intended as of now since I tested it using the old capture controller. Will need to refactor this to the new capture controller.

Reply:
It should be OK to use the same controller actions and routes between the two interfaces -- the functionality is the same, right?